python/list-streams/README.md

```diff
@@ -0,0 +1,68 @@
+# Kinesis Video Streams Counter Lambda Function
+
+This AWS Lambda function fetches the total number of Kinesis Video Streams within a specified AWS region using the AWS SDK for Python (Boto3). It employs a retry mechanism for handling transient errors and provides error handling for potential exceptions.
```

Comment:
Demonstrates how to find the total number of KVS stream resources in any AWS Account Id using the ListStreams API. This helps in monitoring and increasing the total streams limit as needed.